src/main/scala/ch/openolitor/stammdaten/repositories/StammdatenRepositoryQueries.scala

```diff
@@ -1901,7 +1901,7 @@ trait StammdatenRepositoryQueries extends LazyLogging with StammdatenDBMappings
         .leftJoin(abotypMapping as aboTyp).on(depotlieferungAbo.abotypId, aboTyp.id)
         .leftJoin(kundeMapping as kunde).on(depotlieferungAbo.kundeId, kunde.id)
         .leftJoin(personMapping as person).on(kunde.id, person.kundeId)
-        .leftJoin(zusatzAboMapping as zusatzAbo).on(depotlieferungAbo.id, zusatzAbo.hauptAboId)
+        .leftJoin(zusatzAboMapping as zusatzAbo).on(sqls"${depotlieferungAbo.id} = ${zusatzAbo.hauptAboId} and ${zusatzAbo.aktiv} =  true")
```

Comment:
Since Auslieferungen are targeting a specific date, filtering the **currently** active Zusatzabos will lead to a result that depends on the current moment (and not on the date of the Auslieferung).
Wouldn't it be possible to filter the Zusatzabos by the date of delivery based on the validity dates of the Zusatzabo?

Reply:
done